Support database as a backend for audit logs (#67)
paralus/helm-charts: charts/ztka/README.md

| deploy.contour.tls | object | `{}` | TLS properties of the virtual host |
| deploy.elasticsearch.address | string | `""` | Elasticsearch address. Required when `deploy.elasticsearch.enable` is unset. |
| deploy.elasticsearch.enable | bool | `false` | Elasticsearch instance is auto deployed and managed by Helm release when true. |
| deploy.filebeat.enable | bool | `false` | Filebeat is used to collect audit logs into the system via elasticsearch. You can disable this if you don't want audit logs or you want to store audit logs into database. |
| deploy.filebeat.indexPrefix | string | `"ralog"` | You can use this to config the index prefixes for elasticsearch.  This has to match with your filebeat config in `filebeat.daemonset.filebeatConfig.filebeat.yml` |

> | deploy.fluentbit.enable | bool | `true` | of you want to push audit logs to elasticsearch. |

Reviewer: ```suggestion
| deploy.fluentbit.enable | bool | `true` | if you want to push audit logs to database. |
```
